ci/jest-reporter.js

```diff
@@ -56,29 +105,40 @@ function buildDiscordStart(results, runId) {
           name: 'Started at',
           value: startedAt
         },
+        {
+          name: 'Log file (Currently all test run tasks logs running shown, will fix to one relevant one in future)',
+          value: `${logFile}`,
+        },
       ],
     },
   ]
   return discordEmbeds
 }
 
-function buildDiscordSummary(results, runId) {
+function buildDiscordSummary(results, runId, logFileName) {
   let startedAt = results.startTime
   try {
     startedAt = (new Date(results.startTime)).toGMTString()
   } catch {
     // pass
   }
 
-  let title = 'Tests Failed'
+  let title = 'Tests Passed'
   let description = `Run Id: ${runId}`
-  let color = 16711712
-  if (results.numFailedTestSuites < 1) {
-    title = 'Tests Passed'
-    color = 8781568
+  let color = 8781568
+  if (results.numFailedTestSuites > 0) {
```

Comment:
Why do we need to flip the order here from starting with assumption of failure to assumption of success?

Reply:
Yeah, your right it's better the assumption is Failure by default, changing back.